### PR TITLE
11 resources

### PR DESCRIPTION
**Pull request by thomwise**
Check out the code for my resources page. There's a lot in the CSS for the accordion and a little for jquery. Hopefully my naming conventions won't conflict with anything because I did my best to be as specific as possible.

### CODE REVIEW

File: functions.php [hunk 1]
```diff
@@ -119,9 +127,81 @@ function html5blank_header_scripts()
...
+\*------------------------------------*/
+function cssmenumaker_scripts_styles() {  
+   wp_enqueue_style( 'cssmenu-styles', get_template_directory_uri() . '/cssmenu/styles.css');
+   wp_enqueue_script('cssmenu-scripts', get_template_directory_uri() . '/cssmenu/script.js');
+}
+
+add_action('wp_enqueue_scripts', 'cssmenumaker_scripts_styles' );
+
+class CSS_Menu_Maker_Walker extends Walker {
+
```
Comment (RobNelson2989) [inline functions.php:147]: extra space here.

File: js/scripts.js [hunk 2]
```diff
@@ -4,7 +4,32 @@
...
+			  $( '#toggle' ).click(function() {
+				$( this ).toggleClass( "curves" );
+			  });
+			
+			  menu.on('click', function(event) {
+				event.preventDefault();
+
+				var targetParent = $(event.target).parent();
+				if (targetParent.hasClass('has-sub')) {
+				  targetParent.toggleClass('active');
```
Comment (RobNelson2989) [inline js/scripts.js]: Fix the nesting in this area

**Review by RobNelson2989 — APPROVED** [review]
I feel Like this looks great. I only saw one extra space and left a comment. line 146

File: functions.php (hunk 1 above)
Comment (thomwise) [reply functions.php:147]: I'll take care of it. Thanks man.

**Pull request merged**